repos/c2sm/packages/flexpart-cosmo-icon/package.py

```diff
@@ -8,11 +8,11 @@
 from llnl.util.filesystem import working_dir, install_tree
 
 
-class FlexpartCosmo(MakefilePackage):
+class FlexpartCosmoIcon(MakefilePackage):
```

Comment:
Are we still testing COSMO?

Reply:
No but flexpart can deal with COSMO and ICON input.